muskathlon/models/survey_user_input.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
This new file should be imported in __init__.py, otherwise it won't be effective.